File: PYTHON/Program/Anak.py
from Orang import Orang

class Anak(Orang):
    def __init__(self, Nama ="", Usia = 0, Jenis_kelamin= "", Sekolah= "", Cita_Cita =""):
        super().__init__(Nama, Usia, Jenis_kelamin)
        self.Sekolah = Sekolah
        self.Cita_Cita = Cita_Cita
      

    # getter dan setter
    def getSekolah(self):
        return self.Sekolah

    def setSekolah(self, Sekolah):
        self.Sekolah = Sekolah

    def getCitaCita(self):
        return self.Cita_Cita

    def setCitaCita(self, Cita_Cita):
        self.Cita_Cita = Cita_Cita

    #untuk menampilkan data anak
    def print_anak(self):
        print(f"Nama          : {self.nama}")
        print(f"Usia          : {self.usia} tahun")
        print(f"Jenis Kelamin : {self.jenis_kelamin}")
        print(f"Sekolah       : {self.Sekolah}")
        print(f"Cita-cita     : {self.Cita_Cita}")

File: CPP/Program/Anak.cpp
#pragma once
#include <iostream>
#include <string>
#include <vector>
#include "Orang.cpp"
using namespace std;

class Anak : public Orang
{
private:
    string Sekolah;
    string Cita_Cita;

public:
    //construct
    Anak()
    {
    }
    Anak(string nama, int usia, string jenis_kelamin, string sekolah, string cita_cita) : Orang(nama, usia, jenis_kelamin)
    {
        this->Sekolah = sekolah;
        this->Cita_Cita = cita_cita;
    }
    //getter dan setter 
    string getSekolah()
    {
        return Sekolah;
    }

    void setSekolah(string Sekolah)
    {
        this->Sekolah = Sekolah;
    }

    string getCitaCita()
    {
        return Cita_Cita;
    }

    void setCitaCita(string Cita_cita)
    {
        this->Cita_Cita = Cita_Cita;
    }

    //untuk menampilkan data anak
    void printAnak()
    {

        cout << "Nama          : " << getNama() << endl;
        cout << "Usia          : " << getUsia() << " tahun" << endl;
        cout << "Jenis Kelamin : " << getJenis() << endl;
        cout << "Sekolah       : " << getSekolah() << endl;
        cout << "Cita Cita     : " << getCitaCita() << endl;
    }
};